package.json

```diff
@@ -161,7 +161,7 @@
                 },
                 {
                     "command": "vscode-docker.containers.composeGroup.logs",
-                    "when": "never"
+                    "when": "config.docker.containers.groupBy == 'Compose Project Name'"
```

Comment:
This when clause makes it so the command only shows in the palette if containers are grouped by compose project name.